app/frontend/src/components/Datepicker.test.tsx

```diff
@@ -37,17 +29,36 @@ const Form = ({ submitForm }: { submitForm: (data: unknown) => void }) => {
 
 describe("DatePicker", () => {
   it("should submit with proper date for clicking", async () => {
-    const submitForm = jest.fn();
-    render(<Form submitForm={submitForm} />, { wrapper });
+    mockdate.set("2021-03-20");
+    let date: Dayjs | undefined = undefined;
+    render(<Form setDate={(d) => (date = d)} />, { wrapper });
     userEvent.click(screen.getByLabelText(CHANGE_DATE));
     userEvent.click(screen.getByText("23"));
     userEvent.click(screen.getByRole("button", { name: SUBMIT }));
 
     await waitFor(() => {
-      expect(submitForm).toHaveBeenCalledWith({
-        datefield: new Date("2021-03-23"),
-      });
+      expect(date?.format()).toEqual(dayjs("2021-03-23").format());
+    });
+    mockdate.reset();
+  });
+
+  //Doesn't work because it seems impossible to mock the local timezone
+  //so date.format() gives the time in utc
+  it.skip.each`
```

Comment:
Will using this work? https://www.npmjs.com/package/timezone-mock

Reply:
Didn't work, I think because mockdate already replaces the date object. I can try it without setting a mock date, though, but it will be flakey. Or maybe I can fork it and add support for a mock date as well as a mock tz.

Reply:
Probably worth pulling this in and for tests that don't need time zone mocking, mock the relevant Date methods with jest directly? They probably aren't as hard to mock compared to time zones